gocyclo.sh

```diff
@@ -20,4 +20,4 @@ then
     go get github.com/fzipp/gocyclo/cmd/gocyclo
 fi
 
-gocyclo -over 10 -avg .
+gocyclo -over 12 -avg .
```

Comment:
can we just ignore the one function that breaks the rule? (//gocyclo:ignore)

Reply:
it will be heavily refactored after the migration is finished (we will drop support for "jwt" auth type and only expect one token format, so it will be much less complext). I added a reminder to the clean-up task to reduce the gocyclo limit